github/src/main/java/com/novoda/github/reports/service/network/GithubApiService.java

```diff
@@ -71,6 +72,7 @@
             @Query("per_page") int perPageCount
     );
 
+    @Deprecated
```

Comment:
afaik there are two reasons for the deprecation:
1. we're currently not using it
2. it's [under developer preview](https://developer.github.com/v3/issues/timeline/) and due to the work done on reactions it's not actually usable without further work ([see here](https://github.com/novoda/github-reports/commit/73abf35d02055aa2db473a1878076f2bbf734740#diff-2a6f364a11a149c667b01f9862b36e17R7))